packages/ui/src/working-groups/modals/CreateOpening/CreateOpening.tsx

```diff
@@ -0,0 +1,172 @@
+  const context = { hiringTarget: 1, minStake: new BN(50000), group } as OpeningConditions
+  const path = useMemo(() => machineStateConverter(state.value), [state.value])
+  const resolver = useYupValidationResolver<CreateOpeningForm>(OpeningSchema, path)
+  const form = useForm<CreateOpeningForm>({ resolver, mode: 'onChange', defaultValues, context })
+  useEffect(() => {
+    form.trigger(machineStateConverter(state.value) as keyof CreateOpeningForm)
+  }, [machineStateConverter(state.value)])
+
+  const createOpeningTx = useMemo(() => {
+    if (api && group) {
+      const { ...specifics } = form.getValues() as CreateOpeningForm
+      const { description, stakePolicy, rewardPerBlock } = getTxParams(group, specifics)
+      return api.tx[group].addOpening(description, 'Regular', stakePolicy, String(rewardPerBlock))
+    }
+  }, [connectionState, activeMember?.id, form.formState.isValidating, group])
+
+  const { feeInfo } = useTransactionFee(activeMember?.controllerAccount, () => createOpeningTx, [
+    connectionState,
+    createOpeningTx,
+  ])
+  const setExportJsonValue = () => {
+    const { ...specifics } = form.getValues() as CreateOpeningForm
+    const exportValue = {
+      applicationDetails: specifics.durationAndProcess.details,
+      title: specifics.workingGroupAndDescription.title,
+      shortDescription: specifics.workingGroupAndDescription.shortDescription,
+      description: specifics.workingGroupAndDescription.description,
+      applicationFormQuestions: specifics.applicationForm?.questions?.map((question) => {
+        return { question: question.questionField }
+      }),
+      stakingPolicy: {
+        amount: specifics.stakingPolicyAndReward.stakingAmount / Math.pow(10, JOY_DECIMAL_PLACES),
+        unstakingPeriod: specifics.stakingPolicyAndReward.leavingUnstakingPeriod,
+      },
+      rewardPerBlock: specifics.stakingPolicyAndReward.rewardPerBlock / Math.pow(10, JOY_DECIMAL_PLACES),
+    }
+    return exportValue
+  }
+
+  const goToPrevious = useCallback(() => {
+    send('BACK')
+  }, [send])
+
+  useEffect((): any => {
+    if (state.matches('requirementsVerification')) {
+      return feeInfo && send(feeInfo.canAfford ? 'NEXT' : 'FAIL')
+    }
+    if (state.matches('beforeTransaction')) {
+      return feeInfo?.canAfford ? send('NEXT') : send('FAIL')
+    }
+  }, [state, activeMember?.id, feeInfo])
+
+  if (!activeMember || state.matches('requirementsFailed')) {
+    return null
+  }
+
+  if (state.matches('transaction') && createOpeningTx) {
+    const tooltipText = `This adds an opening for ${GroupIdToGroupParam[group]}.`
+    return (
+      <SignTransactionModal
+        additionalTransactionInfo={[{ title: 'Create Opening', tooltipText }]}
+        buttonText="Sign transaction and Create"
+        transaction={createOpeningTx}
+        signer={activeMember.controllerAccount}
+        service={state.children.transaction}
+        useMultiTransaction={{ steps: transactionSteps, active: 1 }}
```

Comment:
I don't think this applies here.

In Pioneer "transaction" means chain transaction. Some flows have multiple transactions: typically like when the user is providing a staking account which has not yet been bound to there membership. But here it can't be the case because the WGL doesn't need to stake anything more than they already staked for their role.

Importing and editing the opening are not chain transactions so they shouldn't be shown as a transaction in the `SignTransactionModal`.